bin/cheritest/cheritest.c

```diff
@@ -981,15 +981,16 @@ static const struct cheri_test cheri_tests[] = {
 	  .ct_desc = "check tags are not stored for tmpfile() MAP_SHARED pages",
 	  .ct_func = cheritest_vm_notag_tmpfile_shared,
 	  .ct_flags = CT_FLAG_SIGNAL | CT_FLAG_SI_CODE | CT_FLAG_SI_TRAPNO,
+#ifdef __riscv
```

Comment:
I'd probably like to alter MIPS to report these as SIGSEGV faults as well.

Reply:
Seems like an OK change to me